### DISCUSSION TITLE
How do i use atom with github?

### DISCUSSION THREAD

**Discussion by EuphonSol** [community/community#23228] [New to GitHub]
<p>So I newly started using atom and GitHub and I noticed I can link them<br>
but I still don’t know how…<br>
neither did any online tutorial help me     :frowning:<br>
I need help plz :beginner:</p>

**Comment by logankilpatrick** [comment]
<p>Atom hooks in with git (and subsequently GitHub) automatically. Simply open a folder/project in atom that links to a git repo and then open the git panel in the bottom right side. Let me know if you have issues with this!</p>

**Comment by logankilpatrick** [comment]
<p>The official link for the GitHub atom integration can be found here: <a href="https://github.atom.io/" rel="noopener nofollow ugc">https://github.atom.io/</a></p>

**Comment by EuphonSol** [comment]
<p>how do i do dat doe<br>
i downloaded both github and git ;-;</p>

**Comment by logankilpatrick** [comment]
<p>Hm, not sure what you mean. The above link should walk you through it all.</p>

**Comment by EuphonSol** [comment]
<p>ok, I guess  ¯_(ツ)_¯<br>
I will check on that<br>
aaand no it doesn’t tell me anything<br>
it just tells me what the GitHub Extention does thats all</p>

**Comment by logankilpatrick** [comment]
<p>Please check out the atom docs linked here: <a href="https://flight-manual.atom.io/using-atom/sections/github-package/" class="inline-onebox" rel="noopener nofollow ugc">GitHub package</a></p>

**Comment by EuphonSol** [comment]
<p>i read that but it still doesnt tell me anything about how do i connect them<br>
i just want to know how to log-in to my github account in the githab package<br>
but i never found anything that usefull</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>You have to open a folder on your computer that was cloned down from GitHub, it will link automatically and use your globally setup git credentials.</p>

**Comment by EuphonSol** [comment]
<p>worked thanks :slightly_smiling_face:</p>
